app/views/users/sessions/new.html.erb

```diff
@@ -29,19 +29,22 @@
   <h3 class="heading"><%= ts('Log in') %></h3>
   <p>
     <strong><%= ts('Reminder:') %></strong>
-    <%= ts("This site is in beta. Things may break or crash without notice.") %>
-    <%= ts("Please report any pesky bugs and") %>
+    <%= ts('This site is in beta. Things may break or crash without notice.') %>
+    <%= ts('Please report any pesky bugs and') %>
     <%= link_to ts('give us your feedback'), feedbacks_path %>!
   </p>
 <% end %>
 
 <div id="loginform">
-  <%= render "user_sessions/passwd" %>
+  <%= render 'users/sessions/form' %>
 </div>
 
 <p>
-  <%= ts("Forgot your password or user name?") %> <%= link_to ts("Reset password"), new_password_path %>
+  <%= ts('Forgot your password or user name?') %> <%= link_to ts('Reset password'), new_password_path(resource_name) %>
   <% if @admin_settings.account_creation_enabled? %>
-    <br \><%= ts("Don't have an account?") %> <%= link_to ts('Create an account now.'), new_user_path %>
+    <br \>
+    <%= ts("Don't have an account?") %>
+    <%= link_to ts('Create an account now.'), new_registration_path(resource_name) %>
   <% end %>
+  <%= link_to "Didn't receive confirmation instructions?", new_confirmation_path(resource_name) %><br />
```

Comment:
I think the `<br />` should be before the link, to make sure it's on a separate line from "Create an account now." The text should also be inside ` ts("")`, and I'm having a problem when I try to follow this link. The error message says: 

```No route matches {:action=>"show", :controller=>"profile", :user_id=>#<User id: nil, created_at: nil, updated_at: nil, email: nil, confirmation_token: nil, login: nil, confirmed_at: nil, encrypted_password: nil, password_salt: nil, suspended: false, banned: false, invitation_id: nil, suspended_until: nil, out_of_invites: true, failed_attempts: nil, reset_password_token: nil, reset_password_sent_at: nil, remember_created_at: nil, sign_in_count: 0, current_sign_in_at: nil, last_sign_in_at: nil, current_sign_in_ip: nil, last_sign_in_ip: nil, confirmation_sent_at: nil, unconfirmed_email: nil, locked_at: nil>}```